frontend/package.json

```diff
@@ -2,18 +2,11 @@
   "name": "trase-frontend",
   "version": "1.0.0",
   "description": "",
-  "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "npm run webpack:server",
-    "dev": "npm run start",
```

Comment:
I was keeping this one as an alias for a "soon to be deprecated" command, but up to you.